prereise/gather/demanddata/nrel_efs/tests/test_get_efs_data.py

```diff
@@ -1,107 +1,138 @@
 import os
+import zipfile
 
 import pandas as pd
 from pandas.testing import assert_frame_equal
 from powersimdata.network.usa_tamu.constants.zones import abv2state
 
 from prereise.gather.demanddata.nrel_efs.get_efs_data import (
+    _check_electrification_scenarios_for_download,
+    _check_path,
+    _check_technology_advancements_for_download,
+    _download_data,
+    _extract_data,
     account_for_leap_year,
-    download_demand_data,
-    download_flexibility_data,
     partition_demand_by_sector,
     partition_flexibility_by_sector,
 )
 
+# Create a dummy demand data set
+cont_states = sorted(set(abv2state) - {"AK", "HI"})
+dummy_demand_data = {
+    "Electrification": ["High"] * 4 * 48 * 8760,
+    "TechnologyAdvancement": ["Rapid"] * 4 * 48 * 8760,
+    "Year": [2030] * 4 * 48 * 8760,
+    "LocalHourID": sorted([i for i in range(1, 8761)] * 4 * 48),
+    "State": sorted([i for i in cont_states] * 4) * 8760,
```

Comment:
Same as above: can `[i for i in cont_states]` be replaced with `list(cont_states)`?

Reply:
Done.